pycbc/workflow/pegasus_workflow.py

```diff
@@ -38,6 +39,9 @@
 PEGASUS_FILE_DIRECTORY = os.path.join(os.path.dirname(__file__),
                                       'pegasus_files')
 
+# Get the logger associated with the Pegasus workflow import
+pegasus_logger = logging.getLogger('Pegasus')
```

Comment:
As you only access this in one place, it doesn't need to be a global variable, just do this above line 325 (unless I'm missing something??)